.gitignore

```diff
@@ -47,3 +47,4 @@ public
 .rbenv-version
 .localeapp/*
 lib/bug_report_templates/tmp
+gemfiles/bin
```

Comment:
What about this? I wouldn't expect any binstubs generated here. Maybe it's related to the old setup with `yarn` and `nvm`?